src/foam/nanos/approval/ApprovableAwareDAO.js

```diff
@@ -403,7 +403,10 @@ foam.CLASS({
           .where(
             foam.mlang.MLang.AND(
               foam.mlang.MLang.EQ(ApprovalRequest.DAO_KEY, "approvableDAO"),
-              foam.mlang.MLang.EQ(ApprovalRequest.OBJ_ID, hashedId),
+              foam.mlang.MLang.OR(
+                foam.mlang.MLang.EQ(ApprovalRequest.OBJ_ID, hashedId),
```

Comment:
Why is the hashedId not modeled?

Reply:
Why does it need to be?

Reply:
On line 397 above, it is better to separate the hashId components with some symbol (like ':').

Reply:
@jlhughes making some more changes to this after some discussion, so hashId will be deprecated by getApprovableCreateKey().
@kgrgreer , will add ":" to the approvablecreatekey hash